pontoon/base/static/js/translate.js

```diff
@@ -1005,6 +1006,10 @@ var Pontoon = (function (my) {
             $('#filter .menu [data-type="missing"], #filter .menu [data-type="fuzzy"], #filter .menu [data-type="suggested"]').addClass('selected');
           }
         }
+
+        if (type == 'status' && self.arrayInArray(filter[type], ['missing', 'fuzzy', 'suggested'])) {
+          $('#filter .menu [data-type="untranslated"]').addClass('selected');
+        }
```

Comment:
This block can be moved outside `markSelectedFilters()`, so it's only exdcuted when neccessary.